ledger/ledger-consensual/src/main/kotlin/net/corda/v5/ledger/consensual/Party.kt

```diff
@@ -0,0 +1,16 @@
+package net.corda.v5.ledger.consensual
+
+import net.corda.v5.base.annotations.CordaSerializable
+import net.corda.v5.base.annotations.DoNotImplement
+import net.corda.v5.base.types.MemberX500Name
+import net.corda.v5.crypto.toStringShort
+import java.security.PublicKey
+
+@DoNotImplement
+@CordaSerializable
+interface Party {
+    val name: MemberX500Name
+    val owningKey: PublicKey
+
+    fun description() = "$name (owningKey = ${owningKey.toStringShort()})"
```

Comment:
Why do we implement things like this as a method/function, rather than as a computed property? i.e.

```
val description: String get() = "$name (owningKey = ${owningKey.toStringShort()})"
```

Reply:
Thanks. I have changed it. (The original came with the DP1 codebase.)